app.py

```diff
@@ -77,10 +77,18 @@ def education():
         data["education"].append(new_education)
         new_education_index = len(data["education"]) - 1
         return jsonify({"id": new_education_index})
+    
+    if request.method == 'PUT':
+        update_data = request.get_json()
+        index = update_data.get("id")
```

Comment:
What if the `id` is not integer?

Reply:
So I can add a condition to check for the above issue and solve the lining issues.